src/components/Search/Search.tsx

```diff
@@ -34,9 +34,9 @@ export const Search = ({
             controlRef={inputRef}
             style={{width}}
             className={b(null, className)}
-            placeholder={placeholder}
             value={value}
             onUpdate={onChange}
+            {...props}
```

Comment:
props spreading may lead to passing of unexpected dependencies - is it possible to use explicit props here?

Reply:
`TextInput` has 26 different props, both `Search` and `DebouncedInput` are just wrappers, that adds some extra logic to `TextInput`. I don't think it's a good idea to pass `TextInput` props explicitly here